addons/website_sale/views/templates.xml

```diff
@@ -1981,26 +1981,42 @@
                             <div class="oe_structure clearfix mt-3" id="oe_structure_website_sale_payment_1"/>
 
                             <t t-if="website_sale_order.amount_total">
-                                <div t-if="providers or tokens" id="payment_method" class="mt-3">
+                                <div class="mt-3">
                                     <h3 class="mb24">Pay with </h3>
+                                </div>
+                                <div t-if="providers or tokens" id="payment_method">
                                     <t t-call="payment.checkout">
                                         <t t-set="footer_template_id" t-value="'website_sale.payment_footer'"/>
                                         <t t-set="submit_button_label">Pay Now</t>
                                     </t>
                                 </div>
                                 <div t-else="" class="alert alert-warning">
                                     <strong>No suitable payment option could be found.</strong><br/>
-                                    If you believe that it is an error, please contact the website administrator.
+                                    <div t-if="not request.env.is_admin()">
+                                        If you believe that it is an error, please contact the website administrator.
+                                    </div>
+                                    <div class="mt-2" groups="base.group_system">
+                                        <a t-if="request.env.company.country_id.is_stripe_supported_country"
+                                           t-attf-href="/web#action=#{action_activate_stripe_id}"
+                                           role="button"
+                                           class="btn btn-primary"
+                                           t-out="'ACTIVATE STRIPE'"
```

Comment:
This string won't be translated :eyes: (same remark for the button below)

cc: @AntoineVDV